package.json

```diff
@@ -26,12 +26,16 @@
     "eslint-config-next": "13.1.6",
     "graphql": "^16.6.0",
     "iron-session": "^6.3.1",
+    "luxon": "^3.3.0",
     "next": "13.1.6",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-hook-form": "^7.43.6",
     "react-hot-toast": "^2.4.0",
     "sass": "^1.58.3",
     "typescript": "4.9.5"
+  },
+  "devDependencies": {
+    "@types/luxon": "^3.3.0"
```

Comment:
This is not necessary, you are already downloading it in the dependencies block

Reply:
The types is not the same as the library itself

Reply:
And are we sure we do not want it in production?

Reply:
Discussed internally, not needed.